build.gradle

```diff
@@ -177,7 +181,11 @@ configure(subprojects.findAll { !it.name.endsWith('-distribution') }) {
 subprojects {
   if (JavaVersion.current().isJava7Compatible()) {
     apply from: rootProject.file('gradle/sign.gradle')
-    apply from: rootProject.file('gradle/deploy.gradle')
+    if (statusIsRelease) {
+      apply from: rootProject.file('gradle/deploy.gradle')
+    } else if (!it.name.endsWith('-distribution') && it.version.endsWith('SNAPSHOT')) {
```

Comment:
I think we should publish distribution snapshots as well as that's the easiest way for early adopters to test the application. Agreed?